extensions/wikia/ArticleComments/classes/ArticleCommentsAjax.class.php

```diff
@@ -207,35 +207,22 @@ static public function axPost() {
 			return $result;
 		}
 
-		if (
-			$title->getNamespace() == NS_USER_TALK &&
-			$response[0] == EditPage::AS_SUCCESS_NEW_ARTICLE &&
-			$title->getText() != $wgUser->getName()
-		) {
-			$user = User::newFromName( $title->getText() );
-
-			if ( $user ) {
-				$user->setNewtalk( true );
```

Comment:
this feature is already implemented elsewhere, it's called: Message Wall & Wall Notifications